### ISSUE TITLE
'???' or "Nothing's playing" shown, but hotkeys work

### ISSUE THREAD

**Issue by WolfganP** [aleab/toastify#82]
If this is a crash, bug or unexpected behaviour report, please use the following template.

## EXPECTED BEHAVIOUR
Name/author of currently playing track to show, and toast to appear on track change

## ACTUAL BEHAVIOUR
'???' or "Nothing's playing" is shown, sometimes the label of previous action (ie Volume ++), no toast appear on track change but hotkeys all work anyways.
It was working on initial installation, stopped working a few days ago (can't attribute it to any Spotify program or Win10 OS update).
There's no obvious errors in the logfile.

## STEPS TO REPRODUCE THE BEHAVIOUR
Click on the bar taskbar icon or press of the Show Toast hotkey

## LOG FILE
*(Drag and drop your `Toastify.log` file here; it can be found in `%LocalAppData%\Toastify`)*

```
13:51:41,220-03:00 INFO  [Toastify.EntryPoint.Main, 61] (1) – Architecture: IntPtr = 64bit, Is64BitProcess = True, Is64BitOS = True
13:51:41,245-03:00 INFO  [Toastify.EntryPoint.Main, 61] (1) – Toastify version = 1.10.8.6157
13:51:42,006-03:00 INFO  [Toastify.Core.Spotify.GetSpotifyPath, 763] (1) – Spotify executable found: "C:\Users\defuser\AppData\Roaming\Spotify\Spotify.exe"
13:51:42,522-03:00 INFO  [Toastify.Core.Spotify.LaunchSpotifyAndWaitForInputIdle, 269] (7) – Launching Spotify...
13:51:42,689-03:00 INFO  [Toastify.Core.Spotify.LaunchSpotifyAndWaitForInputIdle, 286] (7) – Spotify process started with ID 14924
13:51:43,164-03:00 INFO  [Toastify.Core.Spotify.ConnectWithSpotify, 308] (7) – Connecting to Spotify's local APIs endpoint...
16:36:14,105-03:00 INFO  [System.Windows.Application.OnExit, 0] (1) – Toastify terminated with exit code 0.

**Comment by WolfganP** [comment]
Looking at the API project https://github.com/JohnnyCrazy/SpotifyAPI-NET/issues/254, it seems a change in Spotify local API broke many apps/plugins...

**Comment by JohnnyCrazy** [comment]
Yup, local API is kinda down right now, not sure if it will come up again.

@aleab you may wanna consider going full web-api :/

**Comment by ScottBeeson** [comment]
How can Spotify be the best on the market and still suck so much? :(

**Comment by Vankog** [comment]
Well, there is nothing wrong about this step.
The local API was never meant for open access.

**Comment by JohnnyCrazy** [comment]
True, but they also never provided an API for desktop applications. The web api has a lot of functionality, but using it in desktop apps without a Backend is not really possible. (Without leaking keys or similar)

**Comment by aleab** [comment]
Hi folks! I've been away for a while.
Unfortunately I think Toastify will remain dead for some time. When I'll have more time on my hands I'll probably decide whether to find a solution or abandon the project entirely.

**Comment by ScottBeeson** [comment]
Too bad. I literally use(d) this every day.

**Comment by CrazyDavey** [comment]
Agreed -- totally understand, but a huge bummer.  Toastify is so useful.

**Comment by kino90** [comment]
Sad to read this.. Toastify is (was?) indeed a great project.. 😢
